news/426.bugfix

```diff
@@ -0,0 +1,2 @@
+- Fix querying `object_provides` for multiple interfaces using 'and' operator
```

Comment:
Don't include a dash in front of the line in the news file.  `towncrier` will add this automatically when gathering all news snippets.
For the rest it is generally fine.   I would make it double quotes though (for literal/monospace) and add a dot at the end:

```
Fix querying ``object_provides`` for multiple interfaces using 'and' operator.
[fRiSi]
```

Reply:
thanks for your feedback - fixed